recipes/gifsicle/build.sh

```diff
@@ -0,0 +1,4 @@
+autoreconf -i
+./configure --disable-gifview --prefix=${PREFIX}
+make
+make install
```

Comment:
Do you want to run `make check` here too?

Reply:
Assuming it has such a target...

Reply:
I tried `make check` - it works for mac os x but somehow fails for linux.

Reply:
```
./test/testie -p ./src ./test
make: ./test/testie: No such file or directory
make: *** [Makefile:895: check] Error 127
```
Still the files are included in the directory. So I would prefer to leave `make check` for now.

Reply:
You're not concerned the linux build is broken?

Reply:
It complains about a file not found, but the file is there in the repository https://github.com/kohler/gifsicle/blob/master/test/testie so I would assume it depends on the how the tests are executed by `make check`. But as the `help` command is working I am pretty sure the executable is fine as well. Either way I can try to add `make check` later, once the feedstock is available.

Reply:
@beckermr Can we still merge it? Otherwise my suggestion would be to only build the Mac OS X version, because that is where we can run the `make check` tests.